index.html

```diff
@@ -15,176 +15,247 @@
+                    <li>HTML</li>
+                    <li>Css</li>
+                </ul>
+            </div>   
+            <div class="recta">
+                <img src="img/diamond.png" alt="" class="circle">
+                    <h3>Frameworks</h3>
+                <ul class="rect-list">
+                    <li>Bootstrap</li>
+                    <li>Roby on Rails</li>
+                    <li>RSPec</li>
+                    <li>CapyBara</li>
+                    <li>selenium</li>
+                </ul>
             </div>
-        </section>
-        <footer class="contact">
-            <h2>I'm always <br> interested in hearing about new projects, 
-                so if you'd like to chat please get in</h2>
-            <form class="ip-type" action="https://formspree.io/f/xzbokgee" method="post">
-                <input class="Name" type="text" placeholder="    Full Name" maxlength="30" required>
-                <input class="email" type="email" placeholder="   Email address" required>
-                <textarea name="msgt1" class="t1" cols="30" rows="10" maxlength="500" placeholder="  Enter text here" required></textarea> 
-                <button class="get" type="submit">Get in touch</button>
-            </form>
-            <hr class="asch">
-            <div class="ico">
-                <ul class="icons-last">
-                    <li><a href="#"><i class="fa-brands fa-github"></i></a></li>
-                    <li><a href="#"><i class="fa-brands fa-linkedin"></i></a></li>
-                    <li><a href="#"><i class="fa-brands fa-angellist"></i></a></li>
-                    <li><a href="#"><i class="fa-brands fa-twitter"></i></a></li>
-                    <li><a href="#"><i class="fa-solid fa-m"></i></a></li>
+            <div class="recta">
+                    <img src="img/diamond.png" alt="" class="circle">
+                    <h3>Skills</h3>
+                <ul class="rect-list">
+                    <li>Codekitt</li>
+                    <li>Github</li>
+                    <li>Codepen</li>
+                    <li>Gitlab</li>
+                    <li>Terminal</li>
                 </ul>
+            </div> 
+        </div>
+    </section>
+    <footer>
+        <div class="cont">
+            <div class="contact">
+                <p class="cont-dics">I'm always interested in hearing about new projects, 
+                    so if you'd like to chat please get in touch.</p>
+                <form class="ip-type" action="https://formspree.io/f/xzbokgee" method="post">
+                    <input type="text" class="input" placeholder="   Full Name" required>
+                    <input type="email" class="input" placeholder="   Email Address" required>
+                    <textarea name="msgt1" class="input-one" cols="30" rows="10" maxlength="500" placeholder="  Enter text here" required></textarea>
+                    <button type="button" class="cont-btn">Get in touch</button>
+                </form>
```

Comment:
- [ ] There should be one input for Full name this should take the space of both First Name and Last Name inputs (shown in the desktop version). Kindly implement it
- [ ] Kindly decrease the right and left padding of `Get in touch` button to make similar with Figma design
### your design
![contact](https://user-images.githubusercontent.com/81727988/166094024-cfc428a7-295a-48ff-82b7-1f7a5f4078f8.png)
### Figma design
![figmacontact](https://user-images.githubusercontent.com/81727988/166094036-bc763d70-2671-4e13-80eb-deca0928df96.png)